test/functional/p2p_leak_tx.py

```
@@ -23,19 +23,40 @@ def set_test_params(self):
+        self.miniwallet = MiniWallet(self.gen_node)
         # Add enough mature utxos to the wallet, so that all txs spend confirmed coins
-        self.generate(miniwallet, 1)
-        self.generate(gen_node, 100)
+        self.generate(self.miniwallet, 1)
+        self.generate(self.gen_node, 100)
 
-        inbound_peer = self.nodes[0].add_p2p_connection(P2PNode())  # An "attacking" inbound peer
+        self.test_tx_in_block()
+        self.test_notfound_on_unannounced_tx()
+
+    def test_tx_in_block(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_tx_in_block`

Add return type annotation: `None`

(ANN201)